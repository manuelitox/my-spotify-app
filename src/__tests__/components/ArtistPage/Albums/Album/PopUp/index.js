import React from 'react'
import { shallow } from 'enzyme'

import PopUpAlbum from 'components/ArtistPage/Albums/Album/PopUp/'

describe('Component | PopUpAlbum', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <PopUpAlbum 
        title="album title" 
        releaseDate="2016-04-07" 
        numberTracks={Â 10 }
        togglePopUp={ jest.fn() } />
    )
  })
  it('should render a component', () => {
    expect(component).toMatchSnapshot()
  })
})