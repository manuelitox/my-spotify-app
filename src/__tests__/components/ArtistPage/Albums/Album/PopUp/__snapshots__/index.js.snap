// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | PopUpAlbum should render a component 1`] = `
<OnClickOutside(PopUpContainer)
  eventTypes={
    Array [
      "mousedown",
      "touchstart",
    ]
  }
  excludeScrollbar={false}
  outsideClickIgnoreClass="ignore-react-onclickoutside"
  preventDefault={false}
  stopPropagation={false}
  togglePopUp={[MockFunction]}
>
  <styled.div>
    <img
      alt="album title"
      height="300"
      src="placeholderAlbum.svg"
      width="300"
    />
    <h3>
      album title
    </h3>
    <p>
      Total tracks: 
      10
    </p>
    <p>
      Release date: 
      2016-04-07
    </p>
  </styled.div>
  <styled.div>
    <Track
      duration="3 min"
      isPopUpVersion={true}
      name="Empire State of Mind"
      previewUrl="https://p.scdn.co/mp3-preview/d7624ec5f93b6d92c1836a95c40ecce463584f6e?cid=774b29d4f13844c495f206cafdad9c86"
      songIsAvailable={true}
    />
    <Track
      duration="4 min"
      isPopUpVersion={true}
      name="No one"
      previewUrl="https://p.scdn.co/mp3-preview/d7624ec5f93b6d92c1836a95c40ecce463584f6e?cid=774b29d4f13844c495f206cafdad9c86"
      songIsAvailable={true}
    />
    <Track
      duration="3:23 min"
      isPopUpVersion={true}
      name="Girl of Fire"
      previewUrl="https://p.scdn.co/mp3-preview/d7624ec5f93b6d92c1836a95c40ecce463584f6e?cid=774b29d4f13844c495f206cafdad9c86"
      songIsAvailable={true}
    />
    <Track
      duration="5:42 min"
      isPopUpVersion={true}
      name="Fallin'"
      previewUrl="https://p.scdn.co/mp3-preview/d7624ec5f93b6d92c1836a95c40ecce463584f6e?cid=774b29d4f13844c495f206cafdad9c86"
      songIsAvailable={true}
    />
    <Track
      duration="3:23 min"
      isPopUpVersion={true}
      name="Girl of Fire"
      previewUrl="https://p.scdn.co/mp3-preview/d7624ec5f93b6d92c1836a95c40ecce463584f6e?cid=774b29d4f13844c495f206cafdad9c86"
      songIsAvailable={true}
    />
    <Track
      duration="5:42 min"
      isPopUpVersion={true}
      name="Fallin'"
      previewUrl="https://p.scdn.co/mp3-preview/d7624ec5f93b6d92c1836a95c40ecce463584f6e?cid=774b29d4f13844c495f206cafdad9c86"
      songIsAvailable={true}
    />
  </styled.div>
</OnClickOutside(PopUpContainer)>
`;
